File: Answer/1967.作为子字符串出现在单词中的字符串数目.go
/*
 * @lc app=leetcode.cn id=1967 lang=golang
 *
 * [1967] 作为子字符串出现在单词中的字符串数目
 */

// @lc code=start
func numOfStrings(patterns []string, word string) int {
	ans := 0
	for _, p := range patterns {
		if strings.Contains(word, p) {
			ans++
		}
	}
	return ans
}

// @lc code=end

File: Answer/1967.作为子字符串出现在单词中的字符串数目.java
/*
 * @lc app=leetcode.cn id=1967 lang=java
 *
 * [1967] 作为子字符串出现在单词中的字符串数目
 */

// @lc code=start
class Solution {
    public int numOfStrings(String[] patterns, String word) {
        int ans = 0;
        for (String p : patterns) {
            if (word.contains(p)) {
                ans++;
            }
        }
        return ans;
    }
}
// @lc code=end
